go/crd/kube-mail.helmich.me_emailpolicies.yaml

```diff
@@ -0,0 +1,254 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
No matter what, we should _not_ have _two_ differently generated specs for the same CRD in this repo. 😐